bugbug/commit_features.py

```diff
@@ -178,8 +182,21 @@ def __call__(self, commit, **kwargs):
 
 
 class files(object):
+    def __init__(self, min_freq=0.00003):
+        self.min_freq = min_freq
+        self.count = defaultdict(int)
+
+        for commit in repository.get_commits():
```

Comment:
We should only use commits from the training set to calculate the counts (otherwise we leak information from outside the training set into the model).